src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  margin: 10px;
+  width: 200px;
+  flex-direction: column;
+  background-color: $card-color;
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
```

Comment:
The image size and margins are hardcoded. It would be better to make it more flexible, for example by using percentages or flex.